### PR TITLE
[Forms] Remove deprecated property from v0/forms

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description of change

This Lighthouse PR, https://github.com/department-of-veterans-affairs/vets-api/pull/3909, removed a property, `first_issued_on`, that was determined to not be valuable at all, so it's always `null`. We need to update our JSON schema for the va.gov route to reflect this. I could have changed it to allow `null` values, but I just removed it instead to be cleaner.

Error - http://sentry.vfs.va.gov/vets-gov/platform-api-staging/issues/101356/?query=is%3Aunresolved

https://github.com/department-of-veterans-affairs/va.gov-team/issues/6050

## Testing done

Locally ran this API to make sure it wasn't returning a 500 anymore.

## Testing planned

This API route isn't used in production right now, besides in a Drupal draft. We'll test in all environments.

#### Unique to this PR

- [x] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: app/swagger/schemas/forms.rb [hunk 1]
```diff
@@ -38,13 +38,6 @@ class Forms
                 key :example, 'Information for Pre-Complaint Processing'
               end
 
-              property :first_issued_on do
-                key :description, 'The date the form first became available'
-                key :type, :string
-                key :example, '2012-01-01'
-                key :format, 'date'
-              end
-
```
Comment (ncksllvn) [inline app/swagger/schemas/forms.rb]: Should I also update the cassette to reflect this property being removed? It's always `null` from Lighthouse right now.

Comment (johnpaulashenfelter) [reply app/swagger/schemas/forms.rb]: If its part of the response, it should probably stay in the cassette

Comment (ncksllvn) [reply app/swagger/schemas/forms.rb]: Sounds good!

**Comment by ncksllvn** [comment]
Hey @johnpaulashenfelter I think is good to go?

File: lib/forms/schemas/forms.json [hunk 2]
```diff
@@ -28,11 +28,8 @@
               "title": {
                 "type": "string"
               },
-              "first_issued_on": {
-                "type": "string"
-              },
               "last_revision_on": {
-                "type": "string"
+                "type": ["string", "null"]
```
Comment (ncksllvn) [inline lib/forms/schemas/forms.json]: This field is still valid, but now null-able.

File: app/swagger/schemas/forms.rb [hunk 3]
```diff
@@ -38,14 +38,7 @@ class Forms
...
-              property :last_revised_on do
+              property :last_revision_on do
```
Comment (ncksllvn) [inline app/swagger/schemas/forms.rb]: This was an inconsistency I caught between the Swagger docs and the actual response JSON.

**Comment by ncksllvn** [comment]
This can be verified as working via the review instance - http://9eead66401523831f676457729ca0eb6.review.vetsgov-internal/find-va-forms-beta/?q=health

**Pull request merged**